### ISSUE TITLE
xOBSE SetMouseSpeedX unimplemented called from mod 

### ISSUE THREAD

**Issue by Vladim1rZ** [llde/xOBSE#99]
https://www.nexusmods.com/oblivion/mods/36836
When I use mod "Duke Patricks - Combat Archery" with OBSE version 21, functions of the animation of dropping the sight after the shot and raising the sight while pulling the bow work very well.

But when I use xOBSE 22.3.1, these functions do not work and I also get an error in the obse.log journal:
SetMouseSpeedX unimplemented called from mod Duke Patricks - Combat Archery.esp
SetMouseSpeedY unimplemented called from mod Duke Patricks - Combat Archery.esp
[obse.log](https://github.com/llde/xOBSE/files/6642890/obse.log)

**Comment by llde** [comment]
Thanks, I didn't know mods that used that function, so I didn't implement it while rewriting the input system as I could not test it.
Use xOBSE 22.1 for now if this function is important to you.
I will look, but don't know when

**Comment by Vladim1rZ** [comment]
Thanks! Now I use xOBSE version 21.8 and everything works fine.
I will wait maybe you will add this sometime in the next updates.
This mod  "Duke Patricks - Combat Archery" is the best I've ever seen for archery.

**Comment by FeuerTiger86** [comment]
Well, I went back to a previous version before the rewrite, and I don't know if it's supposed to be like this but the mod makes my camera move up or down all the time when a bow is used, even if I'm just holding it without shooting. I tried changing ini settings that are supposed to prevent "jerking" or the bow "lifting up during the draw back of an arrow", but that didn't help. With the current xOBSE version this doesn't happen any more, so maybe the missing functions are bugged in earlier versions. Anyway, I prefer my camera not moving all over the place, so I keep the current version and try to edit the scripts so the error is not spamming all over the log any more.
